packages/evaluation/src/evaluation/evaluator.ts

```diff
@@ -1,24 +1,29 @@
 import { fileURLToPath } from 'url';
-import { dirname, join } from 'path';
+import path, { dirname, join } from 'path';
```

Comment:
Do we want to import `path`? Can we do something like `{ dirname, join, resolve}`?